editor/src/components/canvas/canvas-strategies/strategies/absolute-move-strategy.tsx

```diff
@@ -38,6 +38,15 @@ export function absoluteMoveStrategy(
   const { pathsWereReplaced, paths: retargetedTargets } =
     retargetStrategyToChildrenOfFragmentLikeElements(canvasState)
 
+  if (
+    pathsWereReplaced &&
+    originalTargets.some((originalTarget) =>
+      MetadataUtils.isComponentInstanceFromMetadata(canvasState.startingMetadata, originalTarget),
+    )
+  ) {
+    return null
+  }
```

Comment:
Does this case happen? I'm pretty sure that if it does then this is wrong, as we just shouldn't be replacing the paths at all if we're operating on a component instance. Instead the strategy should be applied to the component instance itself (rather than returning `null`)

Reply:
we should change retargetStrategyToChildrenOfFragmentLikeElements and specifically `getElementFragmentLikeType` to not return `sizeless-div` for components, and these checks then can be removed. I agree that we shouldn't do that as part of this PR, it can either be a follow-up, or a pre-PR.

Reply:
> I agree that we shouldn't do that as part of this PR, it can either be a follow-up, or a pre-PR.

If we do that then this new check above shouldn't exist in any of the strategies as we'll have to immediately remove it anyway.